tools/sigh.ts

```diff
@@ -550,7 +551,7 @@ function buildifier(args: string[]): boolean {
     buildifierOptions.push('--lint=warn', '--mode=check');
   }
 
-  const exclude = /\bnode_modules\b/;
+  const exclude = /node_modules|dist/;
```

Comment:
I think this will match all "dist" substrings, won't it? Might be better to do `/\b(node_modules|dist)\b/`

Reply:
Why not `/^(node_modules|dist)$/`?

Reply:
That might work, depending on whether it matches on the full path, or on each folder name?

Reply:
Line 117 uses \b, fwiw

Reply:
Yes, that `\b` seemed weird to me. 

It tests only the item-name, not the full path. Matching 'not node_modules' seemed like a weird property, if those item-names can even contain whitespace. 

My intent when I removed them was to look for an exact match (which I messed up), still seems to me that's the best bet. WDYT?

Reply:
Using ^ and $ sounds good to me then, thanks!